### PR TITLE
add note for security concerns

### PR DESCRIPTION
**Pull request by BillWagner**
Also, add missing meta-data on new topics.

This is in response to a comment on the .NET Blog entry announcing C# 7.2

[Internal Review URL (private-protected)](https://review.docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/private-protected?branch=pr-en-us-3727)
[Internal Review URL (protected-internal)](https://review.docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/protected-internal?branch=pr-en-us-3727)

### CODE REVIEW

File: docs/csharp/language-reference/keywords/private-protected.md [hunk 1]
```diff
@@ -60,3 +70,4 @@ The `private protected` keyword combination is a member access modifier. A priva
  [public](../../../csharp/language-reference/keywords/public.md)   
  [private](../../../csharp/language-reference/keywords/private.md)   
  [internal](../../../csharp/language-reference/keywords/internal.md)
+ [Security concerns for internal virtual keywords](https://msdn.microsoft.com/library/heyd8kky(v=vs.110))
```
Comment (JRAlexander) [inline docs/csharp/language-reference/keywords/private-protected.md]: The added link is on the same line with the line above. See the internal URL I added.

File: docs/csharp/language-reference/keywords/protected-internal.md [hunk 2]
```diff
@@ -60,3 +70,4 @@ class DerivedClass : BaseClass
  [public](../../../csharp/language-reference/keywords/public.md)   
  [private](../../../csharp/language-reference/keywords/private.md)   
  [internal](../../../csharp/language-reference/keywords/internal.md)
+ [Security concerns for internal virtual keywords](https://msdn.microsoft.com/library/heyd8kky(v=vs.110))
```
Comment (JRAlexander) [inline docs/csharp/language-reference/keywords/protected-internal.md]: Same here.

**Review by JRAlexander — CHANGES_REQUESTED** [review]
Just a few formatting issues.

**Review by JRAlexander — APPROVED** [review]

**Review by Alexandre-FR — COMMENTED** [review]
test

**Comment by mairaw** [comment]
@Alexandre-FR do you need help with anything?

**Pull request merged**